src/ucd-data-fetch.c

```diff
@@ -61,6 +61,7 @@
 struct cloud_struct {
 	char *name;
 	char *ip;
+	char *port;
```

Comment:
Why not make this `int` or even `unsigned int`? And since it's a static table we can just put `80` as value everywhere. Seems strange to have to use `atoi()` everywhere.